### ISSUE TITLE
Try gradient boosting algorithms

### ISSUE THREAD

**Issue by mihaighidoveanu** [mihaighidoveanu/data-science-lab#4]
Following models are more complex (still easy to use) and may come up with better results :
- AdaBoost
- Bagging with SVMs
- GradientBoostingClassifier
- XGBoost

All the models above can be found in Scikit-Learn, except XGBoost(https://machinelearningmastery.com/develop-first-xgboost-model-python-scikit-learn/)
